android/build.gradle

```diff
@@ -30,6 +30,10 @@ if( newDbSize != null && newDbSize.isLong()) {
     dbSizeInMB = newDbSize.toLong()
 }
 
+def useAsyncTaskThreadPoolExecutor = rootProject.hasProperty('AsyncStorage_use_AsyncTaskThreadPoolExecutor') ? rootProject.properties['AsyncStorage_use_AsyncTaskThreadPoolExecutor'] : true
```

Comment:
Should we consider having this disabled by default? @Krizzu?

Reply:
Yes, disabled by default please

Reply:
disable customExecutor or AsyncTaskThreadPoolExecutor by default ?

Reply:
Good point, it's confusing now. To clear it up:

1. Change the name of the property to `AsyncStorage_useCustomExecutor`
2. The default thread pool executor (without setting the property to `true`) should be the one provided from `AsyncTask`, so please change it to:

```
def useCustomExecutor = rootProject.hasProperty('AsyncStorage_useCustomExecutor') 
? rootProject.properties['AsyncStorage_useCustomExecutor'] 
: false
```

3. Use `useCustomExecutor` variable to decide about Executor - `true` to use `SingleThreadedExecutor`, false for `AsyncTask`'s executor